### ISSUE TITLE
[Project 4] get_gps_location permission check 시점 문의

### ISSUE THREAD

**Issue by jnooree** [swsnu/osspr2021#115]
안녕하세요,

`get_gps_location` system call의 permission check 시점과 값을 받아오는 시점 관련 문의입니다.
반드시 permission check 시의 gps 값과 이후 return되는 gps 값이 같아야 하나요?
감사합니다.

**Comment by ooffordable** [comment]
네,  permission check 하는 시점과 get_gps_location을 콜할때 inconsistency 가 없도록 락 등을 이용하여 맞춰주셔야 합니다.

**Comment by wafflejuice** [comment]
안녕하세요. 헷갈리는 부분이 생겨서 질문 드립니다.

Project4.md에 다음과 같은 부분이 있는데
> DO NOT CONFUSE. The only case you need to check GPS accuracy-based permission is when user requests access to file content. In other words, you do not have to consider GPS proximity when implementing sys_get_gps_location, since this syscall requests only file metadata.

get_gps_locatoin system call의 경우, permission check를 할 때 gps location에 따른 permission check는 필요하지 않다고 이해했습니다.
따라서 permission check는 existing file access control mechanism로 충분할 것 같은데, 그렇다면 lock을 사용하지 않아도 inconsistency가 발생하지 않는 것이 아닌지 질문 드립니다.
